src/components/text-area/TextArea.js

```diff
@@ -5,13 +5,16 @@ import uniqueId from 'lodash/uniqueId';
 
 import Label from '../label';
 import Tooltip from '../tooltip';
+import type { Position } from '../tooltip';
```

Comment:
Just a FYI, tooltip's index.ts no longer exports this. It works right now because of a fake index.js.flow file. Ideally this component should be converted to TypeScript, and you can utilize TooltipPosition import.

Reply:
@priyajeet Can that be a follow up PR? or is it now a requirement to convert this component to Typescript

Reply:
Can be followup